benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -99,6 +99,12 @@ void GraphicsBenchmarkApp::Config(ppx::ApplicationSettings& settings)
     settings.grfx.api                   = kApi;
     settings.grfx.enableDebug           = false;
     settings.grfx.swapchain.depthFormat = grfx::FORMAT_D32_FLOAT;
+#if defined(PPX_BUILD_XR)
+    // XR specific settings
+    settings.grfx.pacedFrameRate   = 0;
+    settings.xr.enable             = false; // Change this to true to enable the XR mode
```

Comment:
I am curious on how this will be handled in the future, will you add a flag to enable this? So that you don't have to modify the code to run in XR mode

Reply:
yeah, i think that could be an option.